core/src/lib/tls_openssl_private.cc

```diff
@@ -61,6 +61,7 @@ TlsOpenSslPrivate::TlsOpenSslPrivate()
     , pem_callback_(nullptr)
     , pem_userdata_(nullptr)
     , verify_peer_(false)
+    , config_table_(nullptr)
```

Comment:
we could (and probably should) move all these zero-initializations to the class member's definition in the header.
This is less error-prone as you then cannot forget to initialize a member when you write another constructor. Also when reading the code you won't need to wonder if this member is guaranteed to be zero-initialized or not.

Reply:
agreed and done.